poetry/core/json/schemas/poetry-schema.json

```diff
@@ -559,6 +559,10 @@
                 "secondary": {
                     "type": "boolean",
                     "description": "Declare this repository as secondary, i.e. it will only be looked up last for packages."
+                },
+                "trusted": {
+                    "type": "boolean",
+                    "description": "Declare this repository as trusted, i.e. option --trusted-host will be used when installing packages with pip."
```

Comment:
Lets drop mention of pip here since we longer use pip to download wheels and this needs to be passed into the executor.